includes/discovery/sensors/pre-cache/fabos.inc.php

```diff
@@ -25,3 +25,6 @@
 
 echo 'swSensorEntry ';
 $pre_cache['fabos_sensors'] = snmpwalk_cache_multi_oid($device, 'swSensorEntry', [], 'SW-MIB');
+
+echo 'swSfpStatTable';
+$pre_cache['fabos_Sfp'] = snmpwalk_cache_multi_oid($device, 'swSfpStatEntry', [], 'FA-EXT-MIB');
```

Comment:
Why are you using the pre-cache? you seem to only use this in one sensor?

Reply:
Bot RX and TX sensors are pre-chached but also temp, volt and current. Would you like me to move it to the dbm or should I include the sensors for temp, volt and current ( i don't use those for monitoring)?

Reply:
If you are just using it in one sensor, please just do the snmpwalk there.